bellows/zigbee/zcl/foundation.py

```diff
@@ -101,7 +101,6 @@ class Discrete:
 DATA_TYPE_IDX = {
     t: tidx
     for tidx, (tname, t, ad) in DATA_TYPES.items()
-    if ad is Analog
```

Comment:
Hrm  - I don't recall what part of the spec I put this in for. I'll need some time to checkout where DATA_TYPE_IDX is otherwise used.